rpm/wpa_supplicant-assoc-timeout.patch

```diff
@@ -0,0 +1,16 @@
+diff -up wpa_supplicant-0.7.3/wpa_supplicant/wpa_supplicant.c.assoc-timeout wpa_supplicant-0.7.3/wpa_supplicant/wpa_supplicant.c
+--- wpa_supplicant-0.7.3/wpa_supplicant/wpa_supplicant.c.assoc-timeout	2010-09-07 10:43:39.000000000 -0500
++++ wpa_supplicant-0.7.3/wpa_supplicant/wpa_supplicant.c	2010-12-07 18:57:45.163457000 -0600
```

Comment:
Author: Daniel Williams <dcbw@fedoraproject.org>  2006-03-04 09:24:19
    - Increase association timeout, mainly for drivers that don't fully support
        WPA ioctls yet

Wonder how relevant this is these days even in Fedora context. Doesn't look that intrusive though.

Reply:
We have quite crappy drivers. If this would help in some cases I would at least try. So far all the devices from old to new ones did perform well, some better than before. Might be 2.11 fixes, might be some patches.

Reply:
Further we support some legacy kernels which are much newer than this patch but ancient by todays standards which could justify the patch.

Reply:
Then again so far those ancient kernels, even ones we've already left behind, have been working fine without it or at least nothing popped up as needing investigation.

Reply:
We have had really weird now and then happening issues on a wide range of devices. The ones that were hard to reproduce that even harder to debug. If some of these patches here help with any of those I'd be happy.